Portal/src/Datahub.Portal/Pages/Workspace/WebApp/WorkspaceWebAppInfoTable.razor

```diff
@@ -87,7 +87,7 @@
 
     public void RefreshUrl()
     {
-        _url = $"/{ReverseProxyConfigService.BuildWebAppURL(WorkspaceAcronym)}/";
+        _url = ReverseProxyConfigService.BuildWebAppURL(WorkspaceAcronym);
```

Comment:
I feel when calling this function, we should be explicit in including a boolean.

what do others think

Reply:
was hesitant to do that because forgetting the trailing "/" will break the web apps. I don't see any other use case for not using the trailing except the proxy configuration